NachoSpigot-Server/src/main/java/net/minecraft/server/Explosion.java

```diff
@@ -342,9 +344,9 @@ private void searchForBlocks(it.unimi.dsi.fastutil.longs.LongSet set, Chunk chun
             double stepZ = this.posZ;
 
             for (; f > 0.0F; f -= 0.22500001F) {
-                int floorX = org.bukkit.util.NumberConversions.floor(stepX);
-                int floorY = org.bukkit.util.NumberConversions.floor(stepY);
-                int floorZ = org.bukkit.util.NumberConversions.floor(stepZ);
+                int floorX = (NachoConfig.enableFastMath == true ? FastMath.floorToInt((Double.doubleToRawLongBits(stepX) >>> 63)) : org.bukkit.util.NumberConversions.floor(stepX));
```

Comment:
Optional, but could you remove the ` == true` on all of them? Since they're useless since `enableFastMath` is a boolean anyway.